src/services/meta_transaction_service.ts

```diff
@@ -201,4 +373,21 @@ export class MetaTransactionService {
         };
         return zeroExTransaction;
     }
+    // tslint:disable-next-line: prefer-function-over-method
```

Comment:
issue (non-blocking): we should poll in the background for this every x seconds and cache it. The API can be flakey